pkg/golinters/gocritic/gocritic.go

```diff
@@ -549,10 +551,8 @@ func debugChecksListf(checks []string, format string, args ...any) {
 		return
 	}
 
-	debugf("%s checks (%d): %s", fmt.Sprintf(format, args...), len(checks), sprintSortedStrings(checks))
-}
+	v := slices.Clone(checks)
+	slices.Sort(v)
 
-func sprintSortedStrings(v []string) string {
-	sort.Strings(slices.Clone(v))
-	return fmt.Sprint(v)
+	debugf("%s checks (%d): %s", fmt.Sprintf(format, args...), len(checks), strings.Join(v, ", "))
 }
```

Comment:
Good, you are fixing a bug I identified by working on my PR. 

```go
func sprintSortedStrings(v []string) string { 
         sort.Strings(slices.Clone(v)) 
         return fmt.Sprint(v) 
}
```

`v` was not sorted

I was waiting for a feedback on my PR to report it.